flink-yarn/pom.xml

```diff
@@ -326,6 +334,62 @@ under the License.
 					</execution>
 				</executions>
 			</plugin>
+
+			<!-- Test-jar for unit tests, more information on this:
+                http://stackoverflow.com/questions/1401857/using-maven-to-build-separate-jar-files-for-unit-testing-a-custom-class-loader
+             -->
+			<plugin>
+				<artifactId>maven-assembly-plugin</artifactId>
```

Comment:
Are we doing this only to create a `maven-test.jar` in the `flink-yarn` module? Maybe we could move the respective test to `flink-yarn-tests` where we already create some user code jars.